File: regression/cs/tst12.cs
using Microsoft.Z3;
using System;

class Test
{

    public void Run()
    {        
        using (Context ctx = new Context())
        {
            RealExpr x = ctx.MkRealConst("x");
            RealExpr y = ctx.MkRealConst("y");

            Goal g = ctx.MkGoal();
            g.Assert(ctx.MkGt(x, ctx.MkReal(10)), ctx.MkEq(y, ctx.MkAdd(x, ctx.MkReal(1))));
            g.Assert(ctx.MkGt(y, ctx.MkReal(1)));

            Console.WriteLine(ctx.MkProbe("num-consts").Apply(g));
            Console.WriteLine(ctx.MkProbe("size").Apply(g));
            Console.WriteLine(ctx.MkProbe("num-exprs").Apply(g));
        }
    }
}

File: old-regressions/python/tst12.py

# Copyright (c) 2015 Microsoft Corporation

from z3 import *

x = Real('x')
y = Real('y')
g = Goal()
g.add(x > 10, y == x + 1) 
g.add(y > 1)
print Probe('num-consts')(g)
print Probe('size')(g)
print Probe('num-exprs')(g)
